tests/test_admin.py

```diff
@@ -482,3 +482,49 @@ def test_versions_filters_all_state(self):
             transform=lambda x: x.pk,
             ordered=False,
         )
+
+
+class ContentExpiryCsvExportTestCase(CMSTestCase):
+    def test_export_button_endpoint_response_is_a_csv(self):
+        """
+        Valid csv file is returned when the export endpoint is executes
```

Comment:
This is a bit of a strange wording, maybe something like this works?
```suggestion
        Valid csv file is returned from the admin export endpoint
```